### PR TITLE
Orchestration concepts

### PR DESCRIPTION
**Pull request by giobart**
Working on orchestration concepts section

### CODE REVIEW

**Comment by giobart** [comment]
To correctly render the table in the scheduling section, PR #27 must be merged.

File: content/docs/concepts/high-level-architecture.md [hunk 1]
```diff
@@ -11,78 +11,92 @@ seo:
...
 
-# Oakestra Architecture
-
-As shown in our [getting started](../../getting-started/deploy-your-first-oakestra-cluster/) guide, Oakestra requires 4 key building blocks to operate:
+Oakestra is composed of 3 key building blocks:
 * The [Root Orchestrator](#root-orchestrator)
 * The [Cluster Orchestrator](#cluster-orchestrator)
-* The [Node Engine](#node-engine), and 
-* The [Net Manager](#net-manager)
+* The [Worker Node](#worker-node) composed of **NodeEngine**, and **Net Manager**
```
Comment (nitindermohan) [inline content/docs/concepts/high-level-architecture.md]: Maybe put this in a callout so it stands out?

File: content/docs/concepts/high-level-architecture.md [hunk 2]
```diff
@@ -11,78 +11,94 @@ seo:
   noindex: false # false (default) or true
 ---
 
-# Oakestra Architecture
+{{< callout context="tip" title="Did you know?" icon="outline/rocket" >}}
+Oakestra is composed of 3 key building blocks:
+* The [Root Orchestrator](#root-orchestrator): an orchestrator for clusters. 
+* The [Cluster Orchestrator](#cluster-orchestrator): an orchestrator for workers. 
+* The [Worker Node](#worker-node) composed of **NodeEngine**, and **Net Manager**, manages the workloads.
```
Comment (nitindermohan) [inline content/docs/concepts/high-level-architecture.md]:
```suggestion
* The [Worker Node](#worker-node) machine that executes application microservices.
```

File: content/docs/concepts/high-level-architecture.md [hunk 3]
```diff
@@ -11,78 +11,94 @@ seo:
...
 
 The root orchestrator is the centralized control plane that coordinates the participating clusters.
 
-![](arch-root.png)
+{{< svg "orchestration/arch-root" >}}
 
-The above image describes the components of the root orchestrator. Each component is deployed as a separate container
-and docker-compose is used to integrate and run them.
+The above image describes the components of the root orchestrator. Each component is deployed as a separate service
+and docker compose plugin is used to integrate and run them.
```
Comment (nitindermohan) [inline content/docs/concepts/high-level-architecture.md]:
```suggestion
The image above illustrates the components of the root orchestrator. Each component operates as an independent service, integrated and managed using the Docker Compose plugin.
```

File: content/docs/concepts/scheduling.md [hunk 4]
```diff
@@ -9,4 +9,99 @@ seo:
...
+|Geo|🟠| Under development, the possibility to filter resources based on geographical coordinates.
+|Bandwidth|🟠| Under development, the possibility to filter resources based on data rate limitations.
+|Storage|🟠|It is possible to specify it, but it is not **yet** taken into account by the scheduler 
+|GPU|🟠|Possibility of specifying the GPU cores. But not yet the available GPU drivers. Right now, the support is only for CUDA.
+|TPU|🔴|Not yet under development
+
+
+### Network & Geo constraints
+
+The networking requirements selection and geographic constraints support are coming out in our next release, **v0.5**, and are part of the LDP algorithm update. Stay tuned.
```
Comment (nitindermohan) [inline content/docs/concepts/scheduling.md]: No longer necessary with the updated callout above

**Review by nitindermohan — CHANGES_REQUESTED** [review]
@giobart made some changes directly for clarity. But also left some comments for you to handle

**Review by giobart — COMMENTED** [review]
LGTM, also I integrated your comments.

**Review by nitindermohan — APPROVED** [review]
This looks good now

**Pull request merged**